docs/DeveloperGuide.md

```diff
@@ -772,6 +772,21 @@ testers are expected to do more *exploratory* testing.
    4. Other incorrect delete commands to try: `deletestudent`, `deletestudent x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
+### Setting mastery check result for a student
+
+1. Setting mastery check result for a student while the full student list is being shown
```

Comment:
i think maybe `expanded student list` would be more consistent

Reply:
Ok, will change